### PR TITLE
Expense form page

### PR DESCRIPTION
**Pull request by FNKMYR**

### CODE REVIEW

File: src/components/Expense.js [hunk 1]
```diff
@@ -1,22 +1,69 @@
 import styled from 'styled-components';
+import { Button } from 'semantic-ui-react';
```
Comment (MaggieSchuetz) [inline src/components/Expense.js:2]: Had to look this up - looks very helpful!

File: src/components/ExpenseForm.js [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
       {
         title: inputValue.title,
         amount: inputValue.amount,
+        description: inputValue.description,
+        date: inputValue.date,
       },
     ]);
-    setInputValue({ title: '', amount: '' });
-    inputRef.current.focus(); //This sets the focus back to the title input
+    console.log('Expense has been added');
```
Comment (MaggieSchuetz) [inline src/components/ExpenseForm.js:25]: Did you want this as a console.log or as an alert or something for the user?

File: src/components/ExpenseForm.js [hunk 3]
```diff
@@ -30,26 +38,41 @@ export default function ExpenseForm({ addToExpenses }) {
...
+          maxLength="30"
           placeholder="What did you spend money for?"
           value={inputValue.title}
-          ref={inputRef}
           onChange={event =>
             setInputValue({ ...inputValue, title: event.target.value })
           }
         />
       </label>
+      <label>
```
Comment (MaggieSchuetz) [inline src/components/ExpenseForm.js:50]: One of the coaches remarked this in my code: apparently it's better for accessibility if you don't nest your input in your label, but rather use `<label htmlFor="description"/> <input id="description" .... >` and so on.

File: src/index.js [hunk 4]
```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 import GlobalStyles from './GlobalStyles.js';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
-  <React.StrictMode>
```
Comment (MaggieSchuetz) [inline src/index.js:8]: I think you should leave Strict Mode in, I believe it gives additional checks / warnings.

**Review by MaggieSchuetz — APPROVED** [review]
Nice work! Only a few small remarks (see comments)

File: src/components/ExpenseForm.js (hunk 2 above)
Comment (FNKMYR) [reply src/components/ExpenseForm.js:25]: Put it in to check for myself, can probably remove it now, thanks.

File: src/components/ExpenseForm.js (hunk 3 above)
Comment (FNKMYR) [reply src/components/ExpenseForm.js:50]: Good to know, will edit it accordingly!

**Pull request merged**